i18n/en.json

```diff
@@ -4549,6 +4551,12 @@
   "user.settings.advance.on": "On",
   "user.settings.advance.onForAllMessages": "On for all messages",
   "user.settings.advance.onForCode": "On only for code blocks starting with ```",
+  "user.settings.advance.performance.disableClientPlugins": "Disable Client-side Plugins",
+  "user.settings.advance.performance.info1": "These settings are only intended to be used temporarily while debugging. Remember to turn them off when you're done since they may negatively impact your experience using Mattermost.",
```

Comment:
You may enable these settings temporarily to help isolate performance issues while debugging. It is not recommended to leave these settings enabled for an extended period of time as they can negatively impact your user experience.